README.md

```diff
@@ -102,7 +102,7 @@ configuration package](https://crossplane.io/docs/v1.10/concepts/packages.html)
 so there is a single command to install it:
 
 ```console
-up ctp configuration install xpkg.upbound.io/upbound/platform-ref-aws:v0.5.0
+up ctp configuration install xpkg.upbound.io/upbound/platform-ref-aws:v0.6.0
```

Comment:
Should we also tag the latest version with the `:latest` tag and replace the referenced tag accordingly in the README.md? cc. @jbw976